apm-agent-core/src/main/java/co/elastic/apm/agent/logging/Log4jLoggerFactoryBridge.java

```diff
@@ -35,6 +35,13 @@
     private static final String FQCN = Log4jLoggerFactoryBridge.class.getName();
     private static final String PACKAGE = "co.elastic.apm.agent.sdk.logging";
 
+    @Override
+    public Logger getLogger(String name) {
+        // parent class AbstractLoggerAdapter already 'implements' the method because it has the same signature
+        // this provides an actual implementation of ILoggerFactory while delegating explicitly to the parent impl.
+        return super.getLogger(name);
```

Comment:
[for reviewer] while completely not required, it makes navigation from `ILoggerFactory` class hierarchy more straightforward, otherwise we can miss that the parent class already implements the method.